pkgs/by-name/je/jellyseerr/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  pnpm_9,
+  fetchFromGitHub,
+  stdenv,
+  makeWrapper,
+  nodejs,
+  python3,
+  sqlite,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jellyseerr";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Fallenbagel";
+    repo = "jellyseerr";
+    rev = "v${version}";
+    hash = "sha256-5kaeqhjUy9Lgx4/uFcGRlAo+ROEOdTWc2m49rq8R8Hs=";
+  };
+
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit pname version src;
+    hash = "sha256-xu6DeaBArQmnqEnIgjc1DTZujQebSkjuai9tMHeQWCk=";
+  };
+
+  buildInputs = [ sqlite ];
+
+  nativeBuildInputs = [
+    python3
+    nodejs
+    makeWrapper
+    pnpm_9.configHook
+  ];
+
+  preBuild = ''
+    export npm_config_nodedir=${nodejs}
+    pushd node_modules
+    pnpm rebuild bcrypt sqlite3
+    popd
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+    pnpm build
+    pnpm prune --prod --ignore-scripts
+    rm -rf .next/cache
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share
+    cp -r -t $out/share .next node_modules dist config package.json overseerr-api.yml
```

Comment:
I think we have to copy the `public` directory as well. I see in my testing all static resources return 404s.